tests/Feature/Auth/RegistrationTest.php

```diff
@@ -1,47 +1,67 @@
 <?php
 
+declare(strict_types=1);
+
 namespace Tests\Feature\Auth;
 
+use App\Helpers\SignupHelper;
 use Illuminate\Foundation\Testing\DatabaseTransactions;
-use Laravel\Fortify\Features;
+use Illuminate\Http\Response;
 use Laravel\Jetstream\Jetstream;
-use PHPUnit\Framework\Attributes\Test;
+use Mockery;
 use Tests\TestCase;
 
 class RegistrationTest extends TestCase
 {
     use DatabaseTransactions;
 
-    #[Test]
-    public function registration_screen_can_be_rendered()
+    public function testAccessToRegistrationPage(): void
```

Comment:
Why removing `#[Test]` ?

Reply:
@asbiin, it is enough that the name of the method begins with the word “test” and the phpunit recognizes it as a test method. The attribute is not needed here. see https://docs.phpunit.de/en/11.1/attributes.html#test